recipes/h5pp/all/conanfile.py

```diff
@@ -50,32 +53,36 @@ def config_options(self):
             del self.options.with_spdlog
 
     def requirements(self):
-        self.requires("hdf5/1.12.1")
-        if tools.Version(self.version) < "1.10.0" or self.options.get_safe('with_eigen'):
+        if Version(self.version) < "1.10.0":
+            self.requires("hdf5/1.12.1")
+        else:
+            self.requires("hdf5/1.13.1")
+
+        if Version(self.version) < "1.10.0" or self.options.get_safe('with_eigen'):
             self.requires("eigen/3.4.0")
-        if tools.Version(self.version) < "1.10.0" or self.options.get_safe('with_spdlog'):
-            self.requires("spdlog/1.10.0")
+        if Version(self.version) < "1.10.0" or self.options.get_safe('with_spdlog'):
+            self.requires("spdlog/1.11.0")
 
     def package_id(self):
         self.info.header_only()
```

Comment:
For compatibility with Conan v2, you might want to change`header_only()` to `clear()` here.

```suggestion
        self.info.clear()
```

Reply:
Thanks!